docs/ops/signals/IDFT_7.md

```diff
@@ -0,0 +1,211 @@
+## IDFT <a name="IDFT"></a> {#openvino_docs_ops_signals_IDFT_7}
+
+**Versioned name**: *IDFT-7*
+
+**Category**: Signal processing
+
+**Short description**: *IDFT* layer performs the inverse discrete Fourier transformation of input tensor by specified dimensions.
+
+**Detailed description**: *IDFT* performs the inverse discrete Fourier transformation of input tensor with respect to specified axes.
```

Comment:
Please give more full details and links on reference implementation.

Reply:
If you mean the nGraph reference implementation, then it will be in the separate PR.